src/library/objects/ShowcaseExporter.js

```diff
@@ -0,0 +1,873 @@
+            odd: "#e0e0e0",
+            even: "#f2f2f2"
+        };
+        this.columnCount = 5;
+        this.addName = false;
+        this.loadingCount = 0;
+        this.callback = function () {
+        };
+        this.aircraftTypes = [];
+        this._statsImages = {
+            fp: null,
+            tp: null,
+            aa: null,
+            ar: null,
+            as: null,
+            ev: null,
+            ls: null,
+            dv: null,
+            ht: null,
+            rn: null,
+            or: null
+        };
+        this._equipTypeImages = {};
+        this._shipImages = {};
+        this._otherImages = {};
+        this._equipCanvases = {};
+        this._equipGroupCanvases = {};
+        this._paramToApi = {
+            fp: "api_houg",
+            tp: "api_raig",
+            aa: "api_tyku",
+            ar: "api_souk",
+            as: "api_tais",
+            ev: "api_houk",
+            ls: "api_saku",
+            dv: "api_baku",
+            ht: "api_houm",
+            rn: "api_leng",
+            or: "api_distance"
+        };
+    };
+
+    ShowcaseExporter.prototype._init = function () {
+        this.canvas = document.createElement("CANVAS");
+        this.ctx = this.canvas.getContext("2d");
+        this.allShipGroups = {};
+        this.aircraftTypes = $.unique($.merge(KC3GearManager.carrierBasedAircraftType3Ids, KC3GearManager.landBasedAircraftType3Ids));
+        for (var i in KC3Meta._stype) {
+            if (KC3Meta._stype !== "")
+                this.allShipGroups[i] = [];
+        }
+    };
+
+    ShowcaseExporter.prototype.complete = function () {
+        (this.callback || function () {
+        })();
+    };
+
+    ShowcaseExporter.prototype._drawBorders = function (canvas, ctx, rowWidth) {
+        if (!canvas)
+            canvas = this.canvas;
+        if (!ctx)
+            ctx = this.ctx;
+        if (!rowWidth)
+            rowWidth = this.rowParams.width;
+        for (var x = rowWidth; x < canvas.width; x += rowWidth) {
+            ctx.fillStyle = "#FFF";
+            for (var y = 0; y < canvas.height; y += 20) {
+                ctx.fillRect(x - 1, y, 2, 10);
+            }
+        }
+    };
+
+    ShowcaseExporter.prototype._addCredits = function (canvasData, topLine) {
+        if (!canvasData)
+            canvasData = this.canvas;
+        if (!topLine)
+            topLine = "Ship List";
+
+
+        var canvas = document.createElement("CANVAS");
+        var ctx = canvas.getContext("2d");
+
+        canvas.height = canvasData.height + this.rowParams.height * 2 + 18;
+        canvas.width = canvasData.width;
+
+        ctx.fillStyle = "#FFF";
+        ctx.fillRect(0, 0, canvas.width, canvas.height);
+        ctx.drawImage(canvasData, 0, this.rowParams.height * 2, canvasData.width, canvasData.height);
+
+        var created = "Made in KC3 on " + dateFormat("dd mmm yyyy");
+        ctx.font = "400 14pt \"Helvetica Neue\", Helvetica, Arial, sans-serif";
```

Comment:
as we discuss at discord, create a function to build these font string, and use `px` instead of `pt`.